delegate-app/Main.hs

```diff
@@ -306,23 +344,37 @@ mbQueueAuctionPhases delegateEvents toClientsChannel = do
 -}
 main :: IO ()
 main = do
+  --- XXX: this prevents logging issues in Docker
+  hSetBuffering stdout LineBuffering
+  hSetBuffering stderr LineBuffering
   port <- lookupEnv "PORT"
-
-  let conf :: DelegateServerConfig
-      conf =
-        DelegateServerConfig
-          { host
-          , port = fromMaybe portDefault $ port >>= readMaybe
-          , tick = tick
-          , ping = ping
-          }
-  runWithTracer' tracer $ runDelegateServer conf
+  -- FIXUP: parse actual adress and other params
+  hydraNodeNumber <- read . fromJust <$> lookupEnv "HYDRA_NODE_NUMBER"
+  putStrLn $ "With number: " <> show hydraNodeNumber
+  let actor = case hydraNodeNumber of
```

Comment:
nit: I think it would be nicer to respect the natural ordering of the alphabet (oscar-patricia-rupert)

Reply:
Yes, that is not only non-nice, but probably a reason for previous bug. I forgot to backport fix.